File: AspNetMvcUnleashed/0672329980_source/Chapter03/CS/MvcApplication1/MvcApplication1/Controllers/QuotationController.cs
using System.Collections.Generic;
using System.Web.Mvc;

namespace MvcApplication1.Controllers
{
    public class QuotationController : Controller
    {

        public ActionResult Index()
        {
            return View();
        }


        public ActionResult List()
        {
            var quotes = new List<string>
            {
                "Look before you leap",
                "The early bird gets the worm",
                "All hat, no cattle"
            };
            
            return Json(quotes);
        }

    }
}

File: AspNetMvcUnleashed/0672329980_source/Chapter03/VB/MvcApplication1/MvcApplication1/Controllers/QuotationController.vb
Public Class QuotationController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function


    Function List() As ActionResult
        Dim quotes As New List(Of String)()
        quotes.Add("Look before you leap")
        quotes.Add("The early bird gets the worm")
        quotes.Add("All hat, no cattle")

        Return Json(quotes)
    End Function


End Class
